### PR TITLE
created a nearestFresh_or_Mart

### PR DESCRIPTION
**Pull request by ituageorge**
#### What does this PR do?
<Brief Message Usually Ticket Title>
-it enables user to be able to get the nearest fresh or mart at a given time. 
-Added mart to the index component
-NearestFreshOrMart.js
-NearestFreshMartSection.jsx
-Description of the task to be completed
- still working on it
- Effected the necessary changes to this branch

#### How can this manually be tested?
- clone this repo
```
$ git clone https://github.com/bukkaApp/Mybukka-Frontend
```
- install all dependencies
```
$ npm i
```
- start Application
```
$ npm start:dev
```
- log on to _localhost:PORT_ to interact with created page

#### Relevant PT story(ies)
<PT_STORY_ID>

#### Screenshots (if appropriate)
N/A
#### Questions
N/A

### CODE REVIEW

File: src/features/feed/components/NearestFreshMartSection.jsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+NearestFreshMartSection.propTypes = {
+  mode: PropTypes.string.isRequired,
+  push: PropTypes.func.isRequired,
+  coordinates: PropTypes.arrayOf(PropTypes.number).isRequired,
+  fetchedBukkas: PropTypes.shape({
+    nearbyBukkas: PropTypes.arrayOf(PropTypes.shape({})),
+  }).isRequired,
+  fetchNearbyBukkas: PropTypes.func.isRequired,
+};
```
Comment (akolliy1) [inline src/features/feed/components/NearestFreshMartSection.jsx]: You don't need a new component. Use `OtherComponent.jsx` for both fresh and mart. Remember to adjust it respectively in the index.js file

File: src/features/feed/components/index.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import PropTypes from 'prop-types';
 import AddToCart from 'Components/common/addToCart';
 
 import FoodSection from './FoodSection';
+import NearestFreshMartSection from './NearestFreshMartSection'
```
Comment (akolliy1) [inline src/features/feed/components/index.js]: check the `NearestFreshMartSection.jsx` comment above.

File: src/features/feed/components/index.js [hunk 3]
```diff
@@ -28,6 +30,7 @@ const Feed = ({
     {drink && <OtherSection {...props} push={push} type="drinks" />}
     {search && <SearchResult {...props} push={push} />}
     {fresh && <OtherSection {...props} push={push} type="fresh" />}
+    {mart && <NearestFreshMartSection {...props} push={push} type="mart" />}
```
Comment (akolliy1) [inline src/features/feed/components/index.js]: Also, check  the `NearestFreshMartSection.jsx` comment above.

File: src/features/feed/components/index.js [hunk 4]
```diff
@@ -28,6 +30,7 @@ const Feed = ({
     {drink && <OtherSection {...props} push={push} type="drinks" />}
```
Comment (akolliy1) [inline src/features/feed/components/index.js]: adjust this and its necessary component for mart.

File: src/features/feed/components/index.js [hunk 5]
```diff
@@ -28,6 +30,7 @@ const Feed = ({
...
     {search && <SearchResult {...props} push={push} />}
     {fresh && <OtherSection {...props} push={push} type="fresh" />}
```
Comment (akolliy1) [inline src/features/feed/components/index.js:29]: This is correct but confirm and adjust this component for fresh

File: src/features/feed/components/index.js [hunk 6]
```diff
@@ -36,6 +39,7 @@ const Feed = ({
 export default Feed;
 
 Feed.defaultProps = {
+  mart: false,
   food: false,
   drink: false,
```
Comment (akolliy1) [inline src/features/feed/components/index.js]: remove unused `drink` defaultValue

File: src/features/feed/components/index.js [hunk 7]
```diff
@@ -45,6 +49,7 @@ Feed.defaultProps = {
 };
 
 Feed.propTypes = {
+  mart: PropTypes.bool,
   favorites: PropTypes.bool,
   fresh: PropTypes.bool,
   category: PropTypes.bool,
```
Comment (akolliy1) [inline src/features/feed/components/index.js:51]: Hope you don't have a drink in here too

File: src/features/feed/img/imgLinks.js [hunk 8]
```diff
@@ -6,3 +6,7 @@ export const foodBannerImage =
 
 export const freshBannerImage =
     'https://res.cloudinary.com/deqt3envc/image/upload/v1556741830/samples/Bukka-Images/banner-fresh.png';
+
+export const MartBannerImage =
+    'https://res.cloudinary.com/deqt3envc/image/upload/v1556741830/samples/Bukka-Images/banner-mart.png';
+
```
Comment (akolliy1) [inline src/features/feed/img/imgLinks.js]: Yeah, I almost ignored this. Where is this image coming from? have you uploaded to Cloudinary? Seems you just change the last name of the previous URL from `/banner-fresh.png` to `/banner-mart.png` :)

**Review by akolliy1 — CHANGES_REQUESTED** [review]
Great Job. Adjust this quickly

File: src/features/feed/components/NearestFreshMartSection.jsx (hunk 1 above)
Comment (ituageorge) [reply src/features/feed/components/NearestFreshMartSection.jsx]: New NearestFreshMartSection.js component deleted

**Pull request closed without merging**